packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -1070,6 +1070,43 @@ class _NestedScrollCoordinator implements ScrollActivityDelegate, ScrollHoldCont
     goBallistic(0.0);
   }
 
+  void pointerScroll(double delta) {
+    assert(delta != 0.0);
+
+    delta = -delta;
+
+    goIdle();
+    updateUserScrollDirection(
+        delta > 0.0 ? ScrollDirection.forward : ScrollDirection.reverse);
+    if (_innerPositions.isEmpty) {
+      _outerPosition!.applyClampedPointerSignalUpdate(delta);
+    } else {
+      double innerDelta = delta;
+      // Apply delta to the outer header first if it is configured to float.
+      if (delta > 0.0 && _floatHeaderSlivers)
+        innerDelta = _outerPosition!.applyClampedPointerSignalUpdate(delta);
+
+      // Scroll "up", apply delta to the outerPosition scroll extent.
+      if(delta < 0.0 && innerDelta != 0.0)
+        innerDelta = _outerPosition!.applyClampedPointerSignalUpdate(innerDelta);
+
+      if (innerDelta != 0.0) {
+        double outerDelta = delta > 0.0 ? 0 : innerDelta;
+        for (final _NestedScrollPosition position in _innerPositions) {
+            final double potentialOuterDelta =
+                position.applyClampedPointerSignalUpdate(innerDelta);
+            outerDelta = math.max(outerDelta, potentialOuterDelta);
+        }
+
+        // Scroll "down", Apply the maximum delta of innerPositions to
+        // outerPosition.
+        if (delta > 0.0 && outerDelta != 0.0)
```

Comment:
I will modify this section to include -delta logic.

About -delta since the delta of the mouse wheel is equal to the -delta of darg, -delta would ensure that the applyUserOffset and pointerScroll logics are structured the same and might be easier to understand, not sure what you think?